doc/Doxyfile.in

```diff
@@ -86,8 +86,9 @@ SEARCH_INCLUDES         = YES
 INCLUDE_PATH            = @Range-v3_SOURCE_DIR@/include
 INCLUDE_FILE_PATTERNS   =
 PREDEFINED              = RANGES_DOXYGEN_INVOKED \
-                          CONCEPT_REQUIRES_IMPL_(X)=requires=X \
-                          "CONCEPT_REQUIRES_IMPL(X)=/// \pre X" \
+                          CONCEPT_REQUIRES_=requires= \
```

Comment:
These `CONCEPT_REQUIRES_IMPL` macros aren't actually used anywhere in the code, so I just changed them to their new names.